cli/internal/cache/cache.go

```diff
@@ -24,6 +24,7 @@ type Cache interface {
 	// Fetch returns true if there is a cache it. It is expected to move files
 	// into their correct position as a side effect
 	Fetch(target string, hash string, files []string) (bool, []string, int, error)
+	Assert(hash string) (bool, error)
```

Comment:
Can we call it `Exists`? I think we have some precedence for that term in the codebase, and we don't expect to crash if we fail an assertion here.

Reply:
Perfect - naming things is hard :)